js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -122,24 +111,15 @@ export default function SetupPaidAds() {
 	const disabledComplete = completing === ACTION_SKIP || ! paidAds.isReady;
 
 	function createSkipButton( text ) {
+		const selector = select( STORE_KEY );
+		const billing = selector.getGoogleAdsAccountBillingStatus();
+
 		const eventProps = {
-			opened_paid_ads_setup: 'no',
 			google_ads_account_status: googleAdsAccount?.status,
-			billing_method_status: 'unknown',
-			campaign_form_validation: 'unknown',
+			billing_method_status: billing?.status ?? 'unknown',
```

Comment:
It would be a bit nicer to use `??` because other falsy values, such as `''` or `0`, are not expected in this case.

Reply:
@eason9487 This has been updated to use the logical OR operator now. 👍